spring-integration-jmx/src/main/java/org/springframework/integration/jmx/config/MBeanExporterHelper.java

```diff
@@ -50,10 +54,14 @@
 
 	private final Set<String> siBeanNames = new HashSet<String>();
 
+	private final Set<String> si42ChannelBeanNames = new HashSet<String>();
```

Comment:
Do you have anything in mind to change it in the future ?
I don't like this property name: it isn't flexible